recipes/django-viewflow/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "django-viewflow" %}
+{% set version = "1.0.1" %}
+{% set sha256 = "ae1b4ad831ebda3a4ddac1fd0592b22b5d315cea5f93afd4267ee345d01414ea" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
+    - django-filter >=1.0
```

Comment:
Admittedly the error is vague, but it means that one of these dependencies doesn't exist on Python 3.6. My guess would be `django-filter`. Would recommend raising an issue on that [feedstock]( https://github.com/conda-forge/django-filter-feedstock ) to add Python 3.6 packages.